app/views/sessions/page_requires_login.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>You have access to this page, <%= current_user.username %></</h1>
```

Comment:
What is happening on this page?

Reply:
It's another page to confirm whether the user is logged in or not, probably not necessary moving forward.